apps/meteor/tests/e2e/admin-room.spec.ts

```diff
@@ -0,0 +1,116 @@
+import { faker } from '@faker-js/faker';
+import { Page } from '@playwright/test';
+
+import { Users } from './fixtures/userStates';
+import { createTargetChannel, createTargetPrivateChannel } from './utils';
+import { expect, test } from './utils/test';
+
+test.use({ storageState: Users.admin.state });
+
+test.describe.serial('admin-rooms', () => {
+	let adminPage: Page;
+	let channel: string;
+	let privateRoom: string;
+
+	test.beforeEach(async ({ page }) => {
+		await page.goto('/admin/rooms');
+	});
+
+	test.beforeAll(async ({ browser, api }) => {
+		adminPage = await browser.newPage({ storageState: Users.admin.state });
+		await adminPage.goto('/home');
+		await adminPage.waitForSelector('[data-qa-id="home-header"]');
+
+		channel = await createTargetChannel(api);
+		privateRoom = await createTargetPrivateChannel(api);
+	});
+	test('should display the Rooms Table', async ({ page }) => {
+		await expect(page.locator('[data-qa-type="PageHeader-title"]')).toContainText('Rooms');
+	});
+
+	test('should filter room by name', async ({ page }) => {
+		const input = page.locator('[data-qa-id="AdminRoomSearchInput"]');
```

Comment:
This locator already exists inside `apps/meteor/tests/e2e/page-objects/admin.ts`
You could call it like this after initializing:

```
await admin.inputSearchRooms.click();
await admin.inputSearchRooms.fill(channel);
```